README.md

```diff
@@ -1,3 +1,5 @@
+| readme | [reference](./docs/reference.md)| [faq](./docs/faq.md)  |
```

Comment:
This doesn't render fine. Seems it's bogus markdown table syntax. I can fix and commit to your branch if you want.